packages/components/addon/components/hds/copy/button/index.hbs

```diff
@@ -0,0 +1,17 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+<Hds::Button
+  class="{{if this.isSuccess 'is-success'}}"
```

Comment:
Thanks for updating to use an `Hds::Button`. 

I would add a classname to this: "hds-copy-button" then use that to scope styles which are specific to this component.

I would also use BEM style for this class:
"is-success" => "hds-copy-button--is-success"